Plugins/Bugsnag/Source/Bugsnag/Private/Apple/ApplePlatformBugsnag.cpp

```diff
@@ -25,9 +25,51 @@
 
 DEFINE_PLATFORM_BUGSNAG(FApplePlatformBugsnag);
 
+#if UE_EDITOR
+
+#import <Bugsnag/BSG_KSCrashReportWriter.h>
+#import <BugsnagPrivate/BugsnagConfiguration+Private.h>
+
+#define IS_PIE_WORLD_KEY "isPlayInEditorWorld"
```

Comment:
I wonder if it would be better to use the actual UE property (i.e. with the G prefix) to make it possible to search (in Google / GitHub)?

Reply:
Possibly... this will be visible in the unrealEngine dashboard tab, so I followed the convention used for other properties.